compute/instances/reference-content/understanding-instance-pricing.mdx

```diff
@@ -0,0 +1,91 @@
+- Go to the Instance [creation page](https://console.scaleway.com/instance/servers/create) of the Scaleway console  Note that you must have [an account](/console/my-account/how-to/create-an-account) to access this page.
+- See the dedicated [pricing page](https://www.scaleway.com/en/pricing/?tags=available) of our website.
+
+## Instance billing
+
+- Your hourly billing begins when you [create the Instance](/compute/instances/how-to/create-an-instance/).
+- Billing pauses if you [power off](/compute/instances/how-to/power-off-instance/) the Instance, and recommences when you power it back on.
+- Billing ends when you [delete the Instance](/compute/instances/how-to/create-an-instance/).
+
+You can power off your Instance or delete it at any time: no minimum commitment or notice period is required.
+
+At the end of each calendar month you will be invoiced for your hourly billed usage that month.
+
+## Flexible IP addresses and storage volumes
+
+### Flexible IP address billing 
+
+When you create an Instance, by default a [flexible IP](/compute/instances/concepts/#flexible-ip/) is created for it at the same time. This gives the Instance a public IP address. A flexible IP address is billed separately to the Instance itself, and has its own hourly rate. You can see the hourly rate for a flexible IP address on the Instance [creation page](https://console.scaleway.com/instance/servers/create) of the Scaleway console, or on the dedicated [pricing page](https://www.scaleway.com/en/pricing/?tags=available) of our website.
+
+Your hourly billing for a flexible IP address begins when you reserve the IP address (either during the creation of an Instance, or by creating a [new](/compute/instances/how-to/use-flexips/#how-to-create-a-new-flexible-ip-address) flexible IP address) and ends when you [delete](/compute/instances/how-to/use-flexips/#how-to-delete-a-flexible-ip-address) the address from your account.
+
+<Message type="note">
+  - You can choose to create an Instance without a flexible IP address via the **Advanced settings** option.
+  - Billing of an Instance stops when you power it off, but the Instance's flexible IP remains reserved to your account nonetheless. You will continue to be billed for the flexible IP even if your Instance is powered off, unless you delete the flexible IP.
+  - When you delete an Instance via the console, you are asked whether you want to also delete its flexible IP. If you delete it, billing for this IP will end. If you keep it in your account, billing continues.
+</Message>
+
+You can detach a flexible IP address from your Instance and delete it at any time: no minimum commitment or notice period is required.
+
+At the end of each calendar month, you will be invoiced for your hourly billed flexible IP usage.
```

Comment:
Should we mention that ephemeral IPs are billed in the same way as flexible IPs ? I don't know if many users use them but I know for example Kapsule does.

Reply:
Yeah, I thought maybe we could not mention it since it isn't actually an option you have when you create an Instance in the console (only via the API). It's already pretty complicated...but I could add it to the note, maybe

Reply:
I added a very brief reference